sparkth_mcp/request.py

```diff
@@ -0,0 +1,66 @@
+from enum import Enum
+import json
+import aiohttp
+from .types import LMSError
+
+
+class Auth(str, Enum):
+    JWT = "Jwt"
+    BEARER = "Bearer"
+
+
+async def request(
+    auth: Auth,
+    token: str,
+    http_method: str,
+    url: str,
+    params=None,
+    payload=None,
+    client: aiohttp.ClientSession = None,
+) -> json:
+    if client is None:
+        raise ValueError("ClientSession instance is required.")
+
+    headers = {
+        "Authorization": f"{auth.value} {token}",
+        "Accept": "application/json",
+        "Content-Type": "application/json",
+    }
+
+    async with client.request(
+        http_method, url, headers=headers, params=params, json=payload
+    ) as response:
+        if response.status < 200 or response.status >= 300:
+            print("status = ", response.status)
+
+            raise await handle_error_response(response)
+
+        text = await response.text()
+        if not text.strip():
+            return {}
+
+        try:
+            json_value = await response.json()
+        except Exception:
```

Comment:
`except Exception` is a red flag. We should catch only specific exceptions. 

Same comment below.